css/main.css

```diff
@@ -307,21 +307,134 @@ li {
 }
 
 .list-title {
-  font-family: "Inter", sans-serif;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: bold;
   font-size: 32px;
   line-height: 44px;
 }
 
 .languages {
-  list-style: inside url("../media/svg/rectangle.svg");
+  list-style: inside url('../media/svg/rectangle.svg');
 }
 
 .frameworks {
-  list-style: inside url("../media/svg/square.svg");
+  list-style: inside url('../media/svg/square.svg');
 }
 
 .skills {
-  list-style: inside url("../media/svg/circle.svg");
+  list-style: inside url('../media/svg/circle.svg');
+}
+
+.contact-section {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  margin-bottom: 50px;
+}
+
+.contact-section-head {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.contact-section-info {
+  margin-top: 50px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+  width: 327px;
+  color: #172b4d;
+}
+
+.contact-form {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.fieldset {
+  padding: 0;
+  margin: 0;
+  border: none;
+}
+
+.contact-form input {
+  margin-bottom: 20px;
+  border: none;
+  border-bottom: 1px solid #cfd8dc;
+  outline: none;
+  background-color: white;
+  height: 50px;
+  font-size: 18px;
+  color: #172b4d;
+  padding-left: 22px;
+  width: 90%;
+}
+
+.textarea::placeholder {
+  color: #3c3a39;
+  font-weight: 500;
+}
+
+.textarea {
+  border: 1px solid #cfd8dc;
+  margin-bottom: 20px;
+  width: 90%;
+  height: 170px;
+  padding-left: 22px;
+  padding-top: 25px;
+  font-size: 18px;
+  outline: none;
+  background-color: #fbf8f7;
+}
```

Comment:
- [ ] Please make you stick to the provided design by changing the border style in order to look the same as the one provided in the figma design.
  Kindly, try this: `border: 1px none`;